core/modules/replica/RequestMessenger.h

```diff
@@ -0,0 +1,114 @@
+/*
+ * LSST Data Management System
+ * Copyright 2017 LSST Corporation.
```

Comment:
2018. But, I think you're going to fix/change all the copyright statements in all this new code per Tim's instructions?